core/src/main/resources/lib/hudson/rssBar.jelly

```diff
@@ -24,22 +24,22 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:x="jelly:xml" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <div align="right" style="margin:1em">
-    <a href="${rootURL}/legend">${%Legend}</a>
-    <span style="padding-left:1em">
-      <a href="rssAll"><img src="${imagesURL}/atom.gif" border="0" alt="Feed" height="16" width="16"/></a>
+  <div id="rss-bar">
```

Comment:
@clouless `id` are very helpful when you are identifying high-level UI components or containers, specially, in the layout. And you know for sure that you won't have the same component in the UI twice (since the `id` must be unique).